libs/wingsdk/src/target-tf-azure/app.ts

```diff
@@ -61,5 +86,19 @@ export class App extends CdktfApp implements IApp {
       enumerable: false,
       writable: false,
     });
+
+    // Create a resource group and storage account for all resources in this app
+    this.resourceGroup = new ResourceGroup(this, "ResourceGroup", {
+      location: this.location,
+      name: ResourceNames.generateName(this, RESOURCEGROUP_NAME_OPTS),
+    });
+
+    this.storageAccount = new StorageAccount(this, "StorageAccount", {
```

Comment:
I think it's possible that storage account can be implemented as a singleton, but looking into the azure docs on storage account I think it's nearly always going to be the case we want to create 1 per app. Still pondering this...

Reply:
We could implement this as a lazy property.

Reply:
Yup, agreed was just chatting with @staycoolcall911 about using lazy loading here.